docs/modules/ROOT/pages/tokens.adoc

```diff
@@ -28,5 +28,4 @@ You've probably heard of the ERC20 or ERC721 token standards, and that's why you
 
  * xref:erc20.adoc[ERC20]: the most widespread token standard for fungible assets, albeit somewhat limited by its simplicity.
  * xref:erc721.adoc[ERC721]: the de-facto solution for non-fungible tokens, often used for collectibles and games.
- * xref:erc777.adoc[ERC777]: a richer standard for fungible tokens, enabling new use cases and building on past learnings. Backwards compatible with ERC20.
  * xref:erc1155.adoc[ERC1155]: a novel standard for multi-tokens, allowing for a single contract to represent multiple fungible and non-fungible tokens, along with batched operations for increased gas efficiency.
```

Comment:
A side note but, is this still novel? Might be worth to also have a list of things to upgrade in the docs even if we don't do it for 5.0